### PR TITLE
Add GH project board automation

### PR DESCRIPTION
**Pull request by oskarth**
Now we have a project board for nim-waku, no longer cross-org, so should be simpler. Based on js-waku workflow.

### CODE REVIEW

File: .github/workflows/add-action-project.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add issue to column "New"
+        if: ${{ !!github.event.issue }}
+        uses: alex-page/github-project-automation-plus@v0.6.0
+        with:
+          project: nim-waku
+          column: New
+          repo-token: ${{ secrets.GH_ACTION_PROJECT_MGMT }}
+      - name: Add pull request to column "In Progress"
```
Comment (jm-clius) [inline .github/workflows/add-action-project.yml:20]: Not sure why "In Progress" is not working - perhaps difference in capitalisation?

**Review by jm-clius — APPROVED** [review]
LGTM, except for it not yet working :D. Think it's a minor change though.

**Comment by D4nte** [comment]
> LGTM, except for it not yet working :D. Think it's a minor change though.

It needs to be merged to work

File: .github/workflows/add-action-project.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        if: ${{ !!github.event.pull_request }}
+        uses: alex-page/github-project-automation-plus@v0.6.0
+        with:
+          project: nim-waku
+          column: In Progress
```
Comment (D4nte) [inline .github/workflows/add-action-project.yml:25]:
```suggestion
          column: In progress
```
I'd recommend to use exact same capitalization.

File: .github/workflows/add-action-project.yml (hunk 1 above)
Comment (oskarth) [reply .github/workflows/add-action-project.yml]: Fixed, also changed around a bit

File: .github/workflows/add-action-project.yml (hunk 2 above)
Comment (oskarth) [reply .github/workflows/add-action-project.yml]: Indeed, fixed and also changed a bit

**Pull request merged**